src/frontend/context/fagsak/FagsakContext.tsx

```diff
@@ -34,6 +34,9 @@ const [FagsakProvider, useFagsakContext] = createUseContext(() => {
 
     const [klagebehandlinger, settKlagebehandlinger] = useState<IKlagebehandling[]>([]);
 
+    const [skjemaHarValgtFortroligBarn, settSkjemaHarValgtFortroligBarn] =
```

Comment:
Denne settes kun fra SøknadContext, så hvis man går inn på en url og ikke går via søknadContext, så har man ikke satt denne. 

Burde man i stedet sende med adressebeskyttelseinformasjon i RestUtvidetBehandling?

Reply:
Årsaken til dette er for å kunne sjekke om barn er valgt (huket av - men enda ikke lagt til). Det er nærmeste felles context for søknadskjema og brevmottakermodal..  Ande forslag til hvordan løse dette? 
Den benyttes kun når man er på siden - ellers vil den være false og man sjekker da kun de som er lagt til.

Reply:
Ah sånn ja, hadde ikke skjønt det. Då er det sikkert greit

På en måte er det vel kun kode6-saksbehandlere som har tilgang til å legge til kode6-barn? 
Man skulle kunne velge å fjerne den knappen helt for saksbehandlere med kode6, hvis de likevel ikke skal kunne legge til brevmottakere på saker som har personer med kode6. Då de skal ikke saksbehandle andre behandlinger enn kode6-behandlinger.

(Det burde kanskje være backend-validering i tillegg til dette?)